### ISSUE TITLE
Bug: Error thrown on 'roll' with no dice

### ISSUE THREAD

**Issue by esheyw** [drexl93/roll-tracker#12]
If a roll card for flat damage is put to chat (ie `/r 3[fire]` ) an error is thrown: 
![](https://i.imgur.com/pkkyWbF.png)

**Comment by esheyw** [comment]
just adding a `?` after the `dice[0]` on that line suppresses the error at least. I haven't looked into if that's polluting the roll results yet.

**Comment by drexl93** [comment]
The first issue has been fixed as per your suggestion in the latest release. Thanks for that.

In regards to initiative, it seems to be tracking them fine on my end for the most part. It doesn't seem to track all the rolls when they're made all at once (such as with the Roll All or Roll NPCs button in the combat tracker). I think this is just a function of the way the code is written, where it waits for one roll to be parsed before moving to the next and so it can sometimes miss some rolls. This is also a problem I've noticed with Fortune/Misfortune rolls for a similar reason. I'm certain there's a way to fix it but to be honest I don't really know what that might be at the moment.
